webtau-http/src/main/java/com/twosigma/webtau/http/HttpDataNodes.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2018 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twosigma.webtau.http;
+
+import com.twosigma.webtau.data.traceable.TraceableValue;
+import com.twosigma.webtau.http.datanode.DataNode;
+import com.twosigma.webtau.http.datanode.DataNodeBuilder;
+import com.twosigma.webtau.http.datanode.DataNodeId;
+import com.twosigma.webtau.http.datanode.StructuredDataNode;
+import com.twosigma.webtau.http.validation.HeaderDataNode;
+import com.twosigma.webtau.utils.JsonParseException;
+import com.twosigma.webtau.utils.JsonUtils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+class HttpDataNodes {
+    private static final Function<String, Object> asIs = (v) -> v;
```

Comment:
Isn't this `Function.identity()`?  Or do you specifically need one which returns Object?

Reply:
Never mind, looking at the usage you probably couldn't use Function.identity().